concent_api/verifier/tasks.py

```diff
@@ -12,11 +24,49 @@
 @shared_task
 def blender_verification_order(
     subtask_id:             str,
-    source_package_path:    str,  # pylint: disable=unused-argument
-    result_package_path:    str,  # pylint: disable=unused-argument
+    source_package_path:    str,
+    result_package_path:    str,
     output_format:          Enum,  # pylint: disable=unused-argument
     scene_file:             str,  # pylint: disable=unused-argument
+    report_computed_task:   message.ReportComputedTask,
 ):
+    # Generate a FileTransferToken valid for a download of any file listed in the order.
+    file_transfer_tokens = {}
+
+    # !! Files MUST be added in following order - sources first, them results !!
+    for file_path in (source_package_path, result_package_path):
+        file_transfer_tokens[file_path] = create_file_transfer_token_for_concent(
+            report_computed_task=report_computed_task,
+            operation=message.FileTransferToken.Operation.download,
+            file_path=file_path,
+        )
+
+    # Remove any files from VERIFIER_STORAGE_PATH.
+    clean_directory(settings.VERIFIER_STORAGE_PATH)
+
+    # Download all the files listed in the message from the storage server to local storage.
+    for file_name, file_transfer_token in file_transfer_tokens.items():
+        try:
+            file_transfer_token.sig = None
+            cluster_response = send_request_to_cluster_storage(
```

Comment:
Please don't call it `cluster_storage`. It's 'storage cluster'. If you want to refer to the service that hosts files, please call it a 'storage server'. Cluster storage sounds very confusing to me. When I hear 'cluster storage' I think 'a persistent disk' not a service. I've been seeing this name in various places for a long time now. Please change it if you happen to find it anywhere else too.

Reply:
It is actually in current Concent code so I would like not to mix those two changes into this pull request and make it as a separate issue.

Reply:
But please fix the ones in newly added code at least.

Reply:
But it is the same function, so it has to be renamed everywhere, what I propose to do in a separate issue.

Reply:
I think that it's better to rename it now than let it sit like this because I think I have pointed it out several times before and it's still called "cluster storage" :)

You could just put the commit fixing this at the bottom of your branch and submit it in a separate pull request so that it can go into master faster.

Reply:
:+1: done